get-started/graph-database/graph-database.adoc

```diff
@@ -20,93 +20,89 @@ You should have a basic understanding of what a database is.
 
 toc::[]
 
-=== Overview
+== Overview
 
-Very simply, a graph database is a database designed to treat the relationships between data as a first-class citizen in the data model.  
+Very simply, a graph database is a database designed for storing and retrieving data and the relationships among it.
```

Comment:
Maybe the original sentence is stronger? But I like the additional sentence below.